CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+2.16.0 Release notes
+=============================================================
+### Breaking changes
+* No API breaking changes but the sync protocol is updated to version 26. You will need to upgrade your Realm Object Server to at least version 3.11.0 or use [Realm Cloud](https://cloud.realm.io). If you are using an old Realm Object Server, you will seee error message `Wrong protocol version in Sync HTTP request, client protocol version = 25, server protocol version = 24`. See https://docs.realm.io/platform/using-synced-realms/troubleshoot/version-compatibilities for more information.
```

Comment:
@kneth I believe I reformulated this once and placed the message under Compatibility?

Reply:
Found it. Please use the changelog from here: https://github.com/realm/realm-js/pull/2025 (except for what's no longer included.

Reply:
I might have forgot to do a `git pull` when renaming the branch. I'll update this PR today.